app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -31,89 +54,94 @@
     }
   }
 
-  .publication-header {
-    @include white-links;
-    @include core-19;
-    color: $white;
-
-    background: $light-blue;
-    padding: $gutter $gutter $gutter / 3;
-    margin-bottom: $gutter;
+  .contents-list-wrapper {
```

Comment:
I'd like to use classes more like the grid ones we use elsewhere, eg: `column-quarter-desktop-only`, which would decouple the contents list from the grid. This can come as a later PR.